apps/ui/BaseWindow.cpp

```diff
@@ -406,6 +412,13 @@ void BaseWindow::create(const char* title, const size_t width,
         _screenSpaceProcessor.init(width, height);
 
     _registerKeyboardShortcuts();
+
+    glGenTextures(1, &_fbTexture);
+    glEnable(GL_TEXTURE_2D);
+    glBindTexture(GL_TEXTURE_2D, _fbTexture);
+    glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
```

Comment:
MIN filter not needed?

Reply:
MAG is not needed :)

Reply:
No, no, it's MIN, I'm sure. You are mapping the texture to a bigger area.